### PR TITLE
hub content copy

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/11143

### CODE REVIEW

**Comment by Loquacity** [comment]
This is just a straight copy from https://github.com/uyuni-project/uyuni-docs/pull/185, we'll review/edit in the next PR.

File: modules/large-deployments/pages/hub-api.adoc [hunk 1]
```diff
@@ -1,4 +1,7 @@
 [[lsd-hub-api]]
 = Using the Hub API
 
-Authoritatively enhance covalent services vis-a-vis resource-leveling catalysts for change. Energistically customize integrated materials through adaptive process improvements. Efficiently communicate next-generation niches after one-to-one content. Energistically unleash transparent supply chains vis-a-vis next-generation platforms. Assertively embrace market-driven results vis-a-vis visionary methods of empowerment.
\ No newline at end of file
+hub-xmlrpc-api is a daemon, currently to be launched from the command line.
```
Comment (keichwa) [inline modules/large-deployments/pages/hub-api.adoc:4]:
```suggestion
hub-xmlrpc-api is a daemon, to be launched from the command line.
```

File: modules/large-deployments/pages/hub-api.adoc [hunk 2]
```diff
@@ -1,4 +1,7 @@
...
+
+Once running, you can connect to the hub-xmlrpc-api at port 8888 via any XMLRPC compliant client libraries.
```
Comment (keichwa) [inline modules/large-deployments/pages/hub-api.adoc:6]:
```suggestion
When running, you can connect to the hub-xmlrpc-api at port 8888 via any XMLRPC compliant client libraries.
```

File: modules/large-deployments/pages/hub-api.adoc [hunk 3]
```diff
@@ -1,4 +1,7 @@
...
+Examples section has some examples which could help in order to begin with.
```
Comment (keichwa) [inline modules/large-deployments/pages/hub-api.adoc:7]:
```suggestion
Examples section has some examples.
```

File: modules/large-deployments/pages/hub-auth.adoc [hunk 4]
```diff
@@ -1,4 +1,14 @@
...
 = Hub Authentication Modes
 
-Interactively administrate vertical collaboration and idea-sharing rather than unique scenarios. Conveniently recaptiualize resource sucking expertise whereas pandemic solutions. Competently synergize premier intellectual capital through viral paradigms. Dramatically initiate innovative core competencies through out-of-the-box applications. Completely procrastinate leveraged web-readiness via maintainable resources.
\ No newline at end of file
+Hub supports 3 different authentication modes.
+
+[square]
+* `Manual mode` (default): user needs to provide API credentials for each Server explicitly
+* `Relay mode`: the same credentials used to authenticate against the Hub will be re-used to authenticate Servers. The list of Servers to connect to will still be provided by the user
+* `Auto connect` mode: Hub credentials will be reused for Servers and any Server the user has access to will be automatically connected
```
Comment (keichwa) [inline modules/large-deployments/pages/hub-auth.adoc:9]:
```suggestion
* `Auto connect mode`: Hub credentials will be reused for Servers and any Server the user has access to will be automatically connected
```

**Comment by Loquacity** [comment]
> * consider to remove empty lines now and then.
> * "words" such as hub-xmlrpc-api or part numbers such as 8888 probably deserve asciidoc decoration.
> * I'm not sure whether we want to capitalize Server and Hub.
> 
> This PR is not ready for reviewing ;)

No, that's why I said we're editing in the next PR :wink:

**Pull request merged**